nixos/modules/services/networking/jotta-cli.md

```diff
@@ -0,0 +1,17 @@
+# Jottacloud Command-line Tool {#module-services-jotta-cli}
```

Comment:
I think this doc is a little to short. Can we add some more valuable information?

Reply:
Sure, how about this?

# Jottacloud Command-line Tool {#module-services-jotta-cli}

[Jottacloud](https://jottacloud.com/) is a Norwegian cloud storage provider. Their data centers are located in Norway and are subject to Norwegian and European privacy laws.

The [Jottacloud Command-line Tool](https://docs.jottacloud.com/en/articles/1436834-jottacloud-command-line-tool) is a headless client for interacting with Jottacloud accounts. Features include automatic folder backup, file archival, webhooks, and remote host management.

The Jottacloud Command-line Tool consists of the `jotta-cli` client and the `jottad` daemon. This module adds `jotta-cli` to the environment and starts a user service that runs `jottad`.

## Quick Start {#module-services-jotta-cli-quick-start}

```nix
{
  user.services.jotta-cli.enabled = true;
}
```

This will start `jottad` using default settings. Run `jotta-cli --help` for a list of available commands.

Reply:
In the context of NixOS, this doesn't explain the module in any way and that's what the docs are for. Take a look at nextcloud or matrix-synapse to better get to know what I am talking about. We can also just omit this.

Reply:
What if I remove the doc file, but explain in the `description` of `enable` that this module starts a daemon and makes the client available in system packages?

Reply:
```
enable = mkEnableOption "Jottacloud Command-line Tool. Make jotta-cli available in the environment and start a user service that runs jottad."
```

Reply:
@SuperSandro2000 How about this?

# Jottacloud Command-line Tool {#module-services-jotta-cli}

The [Jottacloud Command-line Tool](https://docs.jottacloud.com/en/articles/1436834-jottacloud-command-line-tool) is a headless [Jottacloud](https://jottacloud.com) client.

## Quick Start {#module-services-jotta-cli-quick-start}

```nix
{
  user.services.jotta-cli.enable = true;
}
```

This adds `jotta-cli` to `environment.systemPackages` and starts a user service that runs `jottad` with the default options.

## Example Configuration {#module-services-jotta-cli-example-configuration}

```nix
user.services.jotta-cli = {
  enable = true;
  options = [ "slow" ];
  package = pkgs.jotta-cli;
};
```

This uses `jotta-cli` and `jottad` from the `pkgs.jotta-cli` package and starts `jottad` in low memory mode.

`jottad` is also added to `environment.systemPackages`, so `jottad --help` can be used to explore options.